IHP/IDE/SchemaDesigner/Controller/Tables.hs

```diff
@@ -37,7 +37,7 @@ instance Controller TablesController where
         render NewTableView { .. }
 
     action CreateTableAction = do
-        let tableName = param "tableName"
+        let tableName = compileIdentifier (param "tableName")
```

Comment:
IMO this should not be happen in the Schema Designer itself, because you can also edit the sql file manually and it should then still work. We should instead put it into the `IHP.SchemaCompiler`